AccountService.js

```diff
@@ -42,26 +50,41 @@ export class AccountService {
     }
   }
 
+  /**
+   * Takes a url and an email then creates an account for a user.
+   * @return {Object}
+   * @memberOf AccountService
+   * @param {Object} config
```

Comment:
After reading the readme produced from this (which is f'ing awesome), I'm not sure using the term `config` is what we want to use here.  I think `options` is the more conventional term for this.

Reply:
@mattcollier give it another look.

Reply:
@aljones15 I see that `config` was changed to `options` in some places.  I think this is much better, are you changing througout?

Reply:
I changed it in the class constructors but not for the functions themselves

Reply:
Will you be changing it in the functions as well?  Changing it throughout is what I am suggesting.

Reply:
changed in the functions however renamed to config in the constructor (because it is assigned to config).